LocalSettings.php

```diff
@@ -1,4 +1,6 @@
 <?php
+require_once "$IP/includes/wikia/Defines.php";
+require_once "$IP/includes/wikia/DefaultSettings.php";
+require_once "$IP/includes/wikia/GlobalFunctions.php";
```

Comment:
I'm wondering if it's possible to include these files only in one place. Currently, I'm seeing them in multiple places which makes it harder to follow.